lesson_21_homework/lib/ui/aplication/adaptive_favorite_widget.dart

```diff
@@ -0,0 +1,136 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_mobx/flutter_mobx.dart';
+import 'package:lesson_21_animations_homework/core/src/model/cocktail_definition.dart';
+import 'package:lesson_21_animations_homework/state/favorites_store.dart';
+import 'package:provider/provider.dart';
+
+class AdaptiveFavoriteWidget extends StatefulWidget {
+  final Color color;
+  final Color backgroundColor;
+  final CocktailDefinition? cocktailDefinition;
```

Comment:
Кажется, может быть не nullable
```suggestion
  final CocktailDefinition cocktailDefinition;
```